spec/registrant_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'spec_helper'
+
+RSpec.describe Registrant do 
+  before(:each) do
+    @registrant_1 = Registrant.new('Bruce', 18, true )
+    @registrant_2 = Registrant.new('Penny', 15 )
+  end
+  
+  describe "Iteration 1" do
```

Comment:
Moving forward, let's not use "Iteration 1" or the like for testing since that's not how we'll see it in the real world. :) Can put in here the method or feature you're testing though.